examples/mysql-WP-pd/README.md

```diff
@@ -0,0 +1,318 @@
+      - name: wordpress
+        image: wordpress
+        ports:
+          - containerPort: 80
+        volumeMounts:
+            # name must match the volume name below
+          - name: wordpress-persistent-storage
+            # mount path within the container
+            mountPath: /var/www/html
+        env:
+          - name: WORDPRESS_DB_PASSWORD
+            # change this
+            value: yourpassword
+    volumes:
+      - name: wordpress-persistent-storage
+        source:
+          # emptyDir: {}
+          persistentDisk:
+            # This GCE PD must already exist and be formatted ext4
+            pdName: wordpress-disk
+            fsType: ext4
+labels:
+  name: wpfrontend
+kind: Pod
+```
+
+Create the pod:
+
+```shell
+$ <kubernetes>/cluster/kubectl.sh create -f wordpress.yaml
+```
+
+And list the pods to check that the status of the new pod changes
+to `Running`.  As above, this might take a minute.
+
+```shell
+$ <kubernetes>/cluster/kubectl.sh get pods
+```
+
+### Start the WordPress service
+
+Once the wordpress pod is running, start its service, specified by `wordpress-service.yaml`. 
+
+The service config file looks like this:
+
+```yaml
+kind: Service
+apiVersion: v1beta1
+id: wpfrontend
+# the port that this service should serve on
+port: 3000
+# just like the selector in the replication controller,
+# but this time it identifies the set of pods to load balance
+# traffic to.
+selector:
+  name: wpfrontend
+# the container on each pod to connect to, can be a name
+# (e.g. 'www') or a number (e.g. 80)
+containerPort: 80
+labels:
+  name: wpfrontend
+createExternalLoadBalancer: true
+```
+
+Note the `createExternalLoadBalancer` setting.  This will set up the wordpress service behind an external IP.
+`createExternalLoadBalancer` only works on GCE.
+Note also that we've set the service port to 3000.  We'll return to that shortly.
+
+Start the service:
+
+```shell
+$ <kubernetes>/cluster/kubectl.sh create -f wordpress-service.yaml
+```
+
+and see it in the list of services:
+
+```shell
+$ <kubernetes>/cluster/kubectl.sh get services
+```
+
+Then, find the external IP for your WordPress service by listing the forwarding rules for your project:
+
+```shell
+$ gcloud compute forwarding-rules list
+```
+
+Look for the rule called `wpfrontend`, which is what we named the wordpress service, and note its IP address.
+
+## Visit your new WordPress blog
+
+To access your new installation, you'll first need to open up port 3000 (the port specified in the wordpress service config) in the firewall. Do this via:
+
+```shell
+$ gcloud compute firewall-rules create wordpress --allow tcp:3000
+```
+
+This will define a firewall rule called `wordpress` that opens port 3000 in the default network for your project.
+
+Now, we can visit the running WordPress app.
+Use the external IP that you obtained above, and visit it on port 3000:
+
+```
+http://<external_ip>:3000
+```
+
+You should see the familiar WordPress init page.
+
```

Comment:
I considered this, but I believe it will cause trouble due to the way "local" volume storage is used for images and plugins.  The replicas can't multiply write to the same PD.